testutils/iotlab.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import re
+
+from iotlabcli.auth import get_user_credentials
+from iotlabcli.rest import Api
+from iotlabcli.experiment import (submit_experiment, wait_experiment,
+                                  stop_experiment, get_experiment,
+                                  exp_resources, AliasNodes)
+
+
+class IoTLABExperiment():
+    """Utility for running iotlab-experiments based on a list of RIOTCtrls
+       expects BOARD or IOTLAB_NODE variable to be set for received nodes"""
+    BOARD_ARCHI_MAP = {
+        'arduino-zero': {'name': 'arduino-zero', 'radio': 'xbee'},
+        'b-l072z-lrwan1': {'name': 'st-lrwan1', 'radio': 'sx1276'},
+        'b-l475e-iot01a': {'name': 'st-iotnode', 'radio': 'multi'},
+        'firefly': {'name': 'firefly'},
+        'frdm-kw41z': {'name': 'frdm-kw41z', 'radio': 'multi'},
+        'iotlab-a8-m3': {'name': 'a8', 'radio': 'at86rf231'},
+        'iotlab-m3': {'name': 'm3', 'radio': 'at86rf231'},
+        'microbit': {'name': 'microbit', 'radio': 'ble'},
+        'nrf51dk': {'name': 'nrf51dk', 'radio': 'ble'},
+        'nrf52dk': {'name': 'nrf52dk', 'radio': 'ble'},
+        'nrf52832-mdk': {'name': 'nrf52832mdk', 'radio': 'ble'},
+        'nrf52840dk': {'name': 'nrf52840dk', 'radio': 'multi'},
+        'nrf52840-mdk': {'name': 'nrf52840mdk', 'radio': 'multi'},
+        'pba-d-01-kw2x': {'name': 'phynode', 'radio': 'kw2xrf'},
+        'samr21-xpro': {'name': 'samr21', 'radio': 'at86rf233'},
+        'samr30-xpro': {'name': 'samr30', 'radio': 'at86rf212b'},
+        'wsn430-v1_3b': {'name': 'wsn430', 'radio': 'cc1101'},
+        'wsn430-v1_4': {'name': 'wsn430', 'radio': 'cc2420'},
```

Comment:
The WSN430 were removed this week from iot-lab and will no longer be available (EOL). So I think those 2 lines could be removed (and the boards removed from RIOT as well)

Reply:
:heavy_check_mark: